scala/ppml/src/test/scala/com/intel/analytics/bigdl/ppml/attestation/DummyAttestationServiceSpec.scala

```diff
@@ -0,0 +1,17 @@
+package com.intel.analytics.bigdl.ppml.attestation
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class DummyAttestationServiceSpec extends FlatSpec with Matchers {
+    "Dummy Attestation " should "work" in {
+        val dummyAttestationService = new DummyAttestationService()
+        val logger = dummyAttestationService.logger
+        val quote = dummyAttestationService.getQuoteFromServer()
```

Comment:
Need to split getQuote and attestWith Server

Reply:
update in https://github.com/intel-analytics/BigDL/pull/4594/files#diff-35e71054734927b078bd0d50242dc3b43f8173e511ccb59b05e4f6e9062e684bR12